smack-extensions/src/main/java/org/jivesoftware/smackx/muc/MultiUserChatManager.java

```diff
@@ -199,6 +206,40 @@ public void processStanza(Stanza packet) {
         };
         connection.addAsyncStanzaListener(invitationPacketListener, INVITATION_FILTER);
 
+        // Listens for all messages that include an XEP-0249 GroupChatInvitation extension and fire the invitation
+        // listeners
+        StanzaListener directInvitationPacketListener = new StanzaListener() {
+            @Override
+            public void processStanza(Stanza packet) {
+                final Message message = (Message) packet;
+                GroupChatInvitation invite =
+                    packet.getExtension(GroupChatInvitation.class);
+
+                if (invite.getRoomAddress() != null) {
+                    EntityBareJid mucJid = JidCreate.entityBareFromOrNull(invite.getRoomAddress());
+                    if (mucJid == null) {
+                        LOGGER.warning("Received invite to non bare JID: '" + message + "'");
+                        return;
+                    }
+                    // Fire event for invitation listeners
+                    final MultiUserChat muc = getMultiUserChat(mucJid);
+                    final XMPPConnection connection = connection();
+                    final MUCUser.Invite mucInvite = null;
```

Comment:
Now that is a problem: `invite` was previously always not null. This has the potential to break apps if they upgrade to a newer Smack version. I think you want to synthesize a `MUCUser.Invite` here.